src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react"
+import styled from "styled-components/native"
+
+import { Fonts } from "../../../data/fonts"
+import Serif from "../../Text/Serif"
+import { Button } from "../Components/Button"
+import { Container } from "../Components/Container"
+import { Timer } from "../Components/Timer"
+import { Title } from "../Components/Title"
+
+interface BidResultProps {
+  winning: boolean
+}
+
+export class BidResult extends React.Component<BidResultProps> {
+  render() {
+    if (this.props.winning) {
+      return (
+        <CenteringContainer>
+          <Icon source={require("../../../../../images/cirlce-check-green.png")} />
+          <Title>You're the highest bidder</Title>
+
+          <TimeLeft>Time left</TimeLeft>
+          <Timer timeLeftInMilliseconds={1000 * 60 * 20} />
+        </CenteringContainer>
+      )
+    } else {
+      return (
+        <Container>
+          <CenteringContainer>
+            <Icon source={require("../../../../../images/circle-x-red.png")} />
+            <Title>Your bid wasn’t high enough</Title>
+
+            <StyledText>Another bidder placed a higher max bid or the same max bid before you did.</StyledText>
+            <StyledText>Increase your max bid to take the lead.</StyledText>
+
+            <TimeLeft>Time left</TimeLeft>
+            <Timer timeLeftInMilliseconds={1000 * 60 * 20} />
+
+            <Divider />
+
+            <Serif>Current bid</Serif>
+            <Title>$45,000</Title>
+          </CenteringContainer>
+
+          <Button text="Bid again" onPress={() => null} />
+        </Container>
+      )
+    }
+  }
+}
+
+const CenteringContainer = styled.View`
+  flex: 1;
+  justify-content: center;
```

Comment:
I spoke to @briansw yesterday afternoon (I meant to post this on the ticket but forgot 😞), but instead of being perfectly centered he suggested we use a percentage (or put it at 50% - <some number of pixels>) in order to have the baseline match what's in zeplin.

Reply:
:+1: good catch. I'll update the PR.